Flutter/calendar/appointments.md

```diff
@@ -721,7 +721,7 @@ You can delete any occurrence, which is an exception from the recurrence pattern
 To add the exception dates in the recurrence series of custom appointment, add the `recurrenceExceptionDates` property to custom class Meeting.
 
 {% tabs %}
-{% highlight Dart %}
+{% highlight dart hl_lines="18 26 65 66 67 68" %}
```

Comment:
Check the line number from 65-68 or 66 - 69

Reply:
Highlighted the proper line.